.github/workflows/Dockerfile.soroban-dev.yml

```diff
@@ -0,0 +1,154 @@
+name: Dockerfile.image-Dockerfile-soroban-dev
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+
+# Prevent more than one build of this workflow for a branch to be running at the
+# same time, and if multiple are queued, only run the latest, cancelling any
+# already running build. The exception being any protected branch, such as
+# master, where a build for every commit will run.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_protected == 'true' && github.sha || github.ref }}
+  cancel-in-progress: true
+
+env:
+  HEAD_SHA: ${{ github.event.pull_request.head.sha || github.sha }}
+  IMAGE: ${{ format('{0}/{1}:{2}', secrets.DOCKERHUB_TOKEN && 'docker.io' || 'ghcr.io', github.repository, github.event_name == 'pull_request' && format('pr-{0}-soroban-dev', github.event.pull_request.number) || 'soroban-dev') }}
+
+jobs:
+  complete:
+    if: always()
+    needs: [build, test]
+    runs-on: ubuntu-latest
+    steps:
+    - if: contains(needs.*.result, 'failure') || contains(needs.*.result, 'cancelled')
+      run: exit 1
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ env.HEAD_SHA }}
+    - uses: docker/setup-buildx-action@5146db6c4d81fbfd508899f851bbb3883a96ff9f
+    - name: Build Quickstart Image
+      run: docker buildx build --no-cache -f Dockerfile.soroban-dev -t $IMAGE --label org.opencontainers.image.revision="${{ env.HEAD_SHA }}" -o type=docker,dest=/tmp/image .
+    - name: Upload Quickstart Image
+      uses: actions/upload-artifact@v2
+      with:
+        name: image-Dockerfile-soroban-dev
+        path: /tmp/image
+
+  test:
+    needs: build
+    strategy:
+      matrix:
+        # TODO when futurenet is up and configured correctly
+        # network: [testnet, pubnet, standalone, futurenet]
+        network: [testnet, pubnet, standalone]
```

Comment:
Should `futurenet` be included here?

Reply:
not yet, as the deployment is not ready per https://github.com/stellar/ops/issues/1832, would be a small follow-on pr to uncomment and enable tests against it when that deployment is ready.